src/shared/utils/billing.js

```diff
@@ -147,6 +147,20 @@ export function useProPlans({ plans }) {
   }
 }
 
+export const findProPlans = ({ plans }) => {
```

Comment:
We normally use `useProPlans` that has a feature flag for enterprise users. It seems repeated to me and not entirely sure what the use case is for it, so I created this new function that we can use in files that don't start with `use<>` as a util